### PR TITLE
fix creating workspace

### PR DESCRIPTION
**Pull request by iraj720**
https://github.com/hcengineering/platform/issues/5140 https://github.com/hcengineering/platform/issues/5139 <https://github.com/hcengineering/platform/issues/5138>

this is a fix for these 3 issues just tested it. its working correct.

also we can add a layer for interpreting mongodb error codes

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjBiZWVkOGU1MGZjYjI5NThiMDViOGQiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.xyAio7fy7od0arkMnlpBv6LszCoH6RjHP5AN7jDQlXY">Huly&reg;: <b>UBERF-6281</b></a></sub>

### CODE REVIEW

File: server/account/src/index.ts [hunk 1]
```diff
@@ -742,52 +737,50 @@ async function generateWorkspaceRecord (
...
-      throw new PlatformError(new Status(Severity.ERROR, platform.status.WorkspaceRateLimit, { workspace }))
+  }
+
+
+  const hashFunc = createHash('md5');
+  hashFunc.update(getEmailName(email) + workspaceName);
+  let Hashedworkspace = hashFunc.digest('hex');
+  const workspace = 'w-' + Hashedworkspace
+  const workspaceUrl = 'w-' + Hashedworkspace + "-" + email
+  await ctx.error('get hash in create workspace', { workspace, workspaceUrl, workspaceName, email })
```
Comment (haiodo) [inline server/account/src/index.ts:761]:
```suggestion
  await ctx.info('get hash in create workspace', { workspace, workspaceUrl, workspaceName, email })
```

**Comment by haiodo** [comment]
Hi @iraj720 thanks for a contribution, could you please rebase and resolve CI issues. thanks.

**Comment by iraj720** [comment]
@haiodo 
yes. Done.

**Comment by iraj720** [comment]
@haiodo closing this one 
the correct one is this
https://github.com/hcengineering/platform/pull/5277

**Pull request closed without merging**